articles/how-to-build-astatic-site-with-gatsbyjs/index.md

```diff
@@ -0,0 +1,272 @@
+- It's easy to set up.
+- It's scalable.
+- It has a great [community of developers](https://www.gatsbyjs.com/contributing/community/).
+
+### Prerequisites
+To get started with GatsbyJs, you will need some background knowledge on the following:
+- [React.js](https://reactjs.org/) (the basics, components, etc.)
+- [Nodejs](https://nodejs.org/en/) (the basics, [npm](https://www.npmjs.com/), etc)
+- [GraphQL](https://graphql.org/) (the basics and advanced)
+
+However there’s no problem if you don’t have any knowledge on these technologies, this article will help you get started with GatsbyJs, so stick around and check the links out later if you are interested in advancing your knowledge.
+
+### Installations
+First of all, you'll need to install [Node.js](https://nodejs.org/en/) on your computer.
+Go to the [official Node.js website](https://nodejs.org/en/) to download the Node.js version for your operating system.
+
+After installing it, open up your terminal and type in:
+
+```bash
+Users/yourname: node –version
+```
+
+and
+
+```bash
+Users/yourname: npm –version
+```
+
+These commands will give you the version of Node.js and node package manager respectively.
+
+Next, in your terminal type in:
+
+```bash
+Users/yourname: npm install –global gatsby-cli
+```
+
+You just installed Gatsby and its command line interface globally in your computer, this is the first step to interact with Gatsby using the command line interface.
+
+In order to confirm you have Gatsby installed, type in:
+
+```bash
+Users/yourname: gatsby –version
+```
+
+### Creating a new GatsbyJs site
+Now let's create a static site using GatsbyJs.
+
+Open up your terminal in your code editor, if you are using [VS Code](https://code.visualstudio.com/), it comes in with a terminal in it. You will be using the GatsbyJs site template from GitHub to create your site from scratch.
+
+Create a **my-first-gatsby-site** folder, and then in your terminal type in
+`Users/first-gatsby-site: gatsby new first-gatsby-site https://github.com/gatsbyjs/gatsby-starter-hello-world`.
+
+To see the new site, you have created, check it out using these commands.
+
+First, go into the file directory you have just created:
+
+```bash
+Users/my-first-gatsby-site: cd first-gatsby-site
+```
+
+Next:
+```bash
+Users/my-first-gatsby-site /first-gatsby-site: npm run develop
+```
+
+You will then get a success message saying *“your site is running at”* :
+`http://localhost:8000`
+
+You can now open up your web browser and access `http://localhost:8000`:
+
+Open `http://localhost:8000` in your browser and you will get a **“hello world”** text display.
+
+Congratulations! You are now a GatsbyJs expert, oops!
+
+I mean almost an expert but that shouldn’t be an issue now because you just created your very own first static site using GatsbyJs.
+
+Now let’s talk about your file directory, it looks overwhelming especially if you are still a beginner using [Nodejs](https://nodejs.org/en/).
```

Comment:
All references to Node.js should be **Node.js**

Reply:
Resolved

Reply:
I revolved this

Reply:
There's still instances of this, @Jethro-magaji. Line 37 and 271.